spec/lib/ruby_smb/smb1/bit_field/nt_create_andx_flags_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe RubySMB::SMB1::BitField::NtCreateAndxFlags do
+  subject(:options) { described_class.new }
+
+  it { is_expected.to respond_to :request_extended_response }
+  it { is_expected.to respond_to :open_target_dir }
+  it { is_expected.to respond_to :request_opbatch }
+  it { is_expected.to respond_to :request_oplock }
+
+  it 'is little endian' do
+    expect(described_class.fields.instance_variable_get(:@hints)[:endian]).to eq :little
+  end
+
+  {
+    :request_extended_response => 0x00000010,
+    :open_target_dir           => 0x00000008,
+    :request_opbatch           => 0x00000004,
+    :request_oplock            => 0x00000002
+  }.each do |field, bitmask|
+    describe "##{field.to_s}" do
```

Comment:
again, please don't do this in rspec, I know it can be tedious to write these tests out, but meta-programming like this in rspec can cause a lot of problems